WooCommerce/WooCommerce.xcodeproj/project.pbxproj

```diff
@@ -5095,7 +5095,7 @@
 				};
 			};
 			buildConfigurationList = B56DB3C12049BFAA00D4AA8E /* Build configuration list for PBXProject "WooCommerce" */;
-			compatibilityVersion = "Xcode 8.0";
+			compatibilityVersion = "Xcode 11.0";
```

Comment:
This is so that CocoaPods 1.10 continues to use `inputFileList` and `outputFileList` features of Script Build Phases, because now CocoaPods 1.10 rightfully checks for this `compatibilityVersion` now to know if the feature is available (while it based that decision on the project's `objectVersion` before).

So if we kept `Xcode 8.0` as the project format / compatibilityVersion, CocoaPods would revert back to using the `inputFiles` and `outputFiles` property of Script Build Phases instead of taking advantage of `xcfilelists`, which would (1) not only make that diff way bigger (2) but also lose us the advantage of the project only referencing these `xcfilelist` – which means the `pbxproj` don't need to be changed and generate a diff if pods are changed and the script build phase requires to depend on different input/output files)